docs/reference/modules/scripting/security.asciidoc

```diff
@@ -1,67 +1,62 @@
 [[modules-scripting-security]]
 === Scripting and security
 
-You should never run Elasticsearch as the `root` user, as this would allow a
-script to access or do *anything* on your server, without limitations.
-
-You should not expose Elasticsearch directly to users, but instead have a
-proxy application inbetween. If you *do* intend to expose Elasticsearch
-directly to your users, then you have to decide whether you trust them enough
-to run scripts on your box or not, and apply the appropriate safety measures.
-
-[[enable-dynamic-scripting]]
```

Comment:
Do you think I should keep the anchor somewhere with a comment that it is historical? Something about how it is no longer needed in most cases?